packages/sui-bundler/README.md

```diff
@@ -167,6 +167,8 @@ This tool works with zero configuration out the box but you could use some confi
 
 `targets`: Object with information about the browser and version supported. (default: `see the next example`)
 
+`disableTSLoader`: Flag to determine if typescript loader should be used. (default: `false`)
```

Comment:
I would try to use the same name regarding "TypeScript", so in this case I'd rather use `disableTypeScriptLoader`, if not, we'll have different names for the same thing along all the repo code.

Reply:
sounds good